pkg/systemd/services/services.jsx

```diff
@@ -536,17 +536,15 @@ class ServicesPage extends React.Component {
             unit.CombinedState = _("Not running");
 
         unit.AutomaticStartup = "";
+        if (unit.UnitFileState && unit.UnitFileState.length > 1) {
+            unit.AutomaticStartup = cockpit.format(_("%0", unit.UnitFileState.charAt(0).toUpperCase() + unit.UnitFileState.slice(1)));
+        }
         if (unit.UnitFileState && unit.UnitFileState.indexOf('enabled') == 0) {
-            unit.AutomaticStartup = _("Enabled");
```

Comment:
I'm afraid these explicit strings were deliberate -- with this current change, they stop being translatable. Also, IMHO showing "Enabled-runtime" is a bit confusing. Either we make proper human strings, or we don't translate them at all and show them verbatim, but then we also shouldn't show them with upper case.

How about a compromise? Keep the four "common" states that we nd, and if it's none of them, show the file state verbatim?

Reply:
Indeed, I came here to comment on the same.
A few days ago I opened this PR: https://github.com/cockpit-project/cockpit/pull/13896
That was just very ugly way of doing things, but I did this because Japanese users were not super happy about random English strings in otherwise completely translated page.

I think it would be the best, if we could actually make the set of supported states bigger and make sure they are properly translated (and also capitalized). And if the state is not supported, maybe just omit it? (see the list in my PR, the possibilities are huge to have everything).

Or have label like 'Other' and that would have tooltip with the real state? (or just have some generic one like 'Not running' with tooltip showing the real state - in english)

Reply:
I think having these 12 states pretranslated as you did in your PR is not such an issue.
And have an exception for the enabled-runtime masked-runtime, we can just show Enabled/Masked for these two.

Reply:
ah right, true.
Then I would go with really enumerating all of the options so they are translated. And showing only `foo` from `foo-runtime`. In that case it is just 9 states, out of which half is currently used anyway.